package.json

```diff
@@ -30,5 +30,11 @@
     "url": "http://www.bentley.com"
   },
   "//dependencies": "NOTE: dependencies will be empty here and instead specified in each package.",
-  "//devDependencies": "NOTE: devDependencies will be empty here and instead specified in each package."
+  "//devDependencies": "NOTE: devDependencies will be empty here and instead specified in each package.",
+  "dependencies": {
+    "@storybook/preview-api": "^8.0.0"
```

Comment:
Revert these changes, these dependencies should only be specified in `docs/storybook/package.json`.